docs/source-app/examples/file_server/app.py

```diff
@@ -3,9 +3,7 @@
 import tarfile
 import uuid
 import zipfile
-from dataclasses import dataclass
```

Comment:
Let's not add noqa in this file. This is a doc file and users don't want to see them.

Reply:
if you do not use noqua, follow python standards ;)